theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/Row.tsx

```diff
@@ -26,6 +33,17 @@ const NodeWrapper = styled.div<{isEven: boolean}>`
     box-sizing: border-box;
     border-bottom: 1px solid #252b3869;
     background: ${(props) => (props.isEven ? 'transparent' : '#6b8fb505')};
+    transition: background 0.15s ease-in-out;
+  }
+
+  &:hover:before {
+    transition: background 0.05s ease-in-out;
+    background: ${(props) =>
+      props.canHighlight
+        ? '#7a22221f'
```

Comment:
I just made this colour up 

<img width="1187" alt="image" src="https://user-images.githubusercontent.com/1253802/152961142-410e34b9-cb99-4f75-90c3-223254252e58.png">